tutorials/source-modeling/plot_source_space_SNR.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+"""
+.. _tut-mne-sourcespace-snr:
+
+===============================
+Computing source space SNR
+===============================
+
+This example shows how to compute and plot source space SNR.
+
+Reference:
+Goldenholz, D. M., Ahlfors, S. P., Hämäläinen, M. S., Sharon, D., Ishitobi, M.,
+Vaina, L. M., & Stufflebeam, S. M. (2009). Mapping the Signal-To-Noise-Ratios
+of Cortical Sources in Magnetoencephalography and Electroencephalography.
+Human Brain Mapping, 30(4), 1077–1086. doi:10.1002/hbm.20571
+
+"""
+# Author: Padma Sundaram <tottochan@gmail.com>
+#         Kaisu Lankinen <klankinen@mgh.harvard.edu>
+#
+# License: BSD (3-clause)
+
```

Comment:
Looks reasonable so far

https://15977-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/source-modeling/plot_source_space_SNR.html

But I would add `# sphinx_gallery_thumbnail_number = 2` up here somewhere to improve the thumbnail